app/presenters/summary/sections/justification_for_legal_aid.rb

```diff
@@ -0,0 +1,36 @@
+module Summary
+  module Sections
+    class JustificationForLegalAid < Sections::BaseSection
+      def name
+        :justification_for_legal_aid
+      end
+
+      def show?
+        kase.present? && super
```

Comment:
We will also need to protect against a potential nil `ioj`, just in case.

Reply:
Why is that needed here but not in https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/blob/main/app/presenters/summary/sections/next_court_hearing.rb

Reply:
Because on the NextCourtHearing we only access "kase" (so we protect against it being `nil`).

But in the IoJ we have a 2-level relationship where to reach `ioj` you need to go through `case`. Case could be present but `ioj` be nil and then in the loop the code will blow up 😝

Reply:
Gotcha, ok, thank you!